scrapy/http/request/form.py

```diff
@@ -170,10 +170,12 @@ def _get_clickable(clickdata, form):
     """
     clickables = [
         el for el in form.xpath(
-            'descendant::*[(self::input or self::button)'
-            ' and re:test(@type, "^submit$", "i")]'
-            '|descendant::button[not(@type)]',
-            namespaces={"re": "http://exslt.org/regular-expressions"})
```

Comment:
is it me or these lines are changed from 4 spaces indentation to 5 spaces?

Reply:
TL;DR it would be much easier to review if you keep the indentation at 4 spaces.

Reply:
Sure, I will make the necessary changes.

Reply:
@dangra @cathalgarvey Resolved this review. Please see the new changes.